apps/i18n/common/en_us.json

```diff
@@ -727,6 +729,7 @@
   "languages": "Languages",
   "lastEdited": "Last Edited",
   "lastUpdated": "Last updated {time}",
+  "lastUpdatedNoTime": "Last updated:",
```

Comment:
teeny tiny nit: should this be "Last Updated:" ? The other strings had both words in the phrase capitalized, but I don't know if we have consensus on this (or if anyone else notices)

Reply:
I have no idea what the rules are around this. Anyone know?

Reply:
Maybe it's a product question?  Or maybe we don't care?  I'd just expect us to be consistent one way or the other...

Reply:
we usually use title case ("Last Updated:"). this string should also include the timestamp so that it's easier for translation/RTL:

```
"lastUpdated:" "Last Updated: {time}",
```

(i know the key above currently won't work since it already exists on line 735)

Reply:
ignore my comment above about adding `{time}` to the string -- i noticed below that time is on a separate line, so it doesn't make sense to condense the two!

Reply:
I think putting the time on the same line will make it harder to format but I will change it to be all upper case.